react/react1/week1/todo/src/components/TodoList.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ListItem from "./ListItem";
+
+export default function TodoList({ tasks }) {
+  console.log("%cTodoList.jsx line:5 tasks", "color: #007acc;", tasks);
+  return (
+    <ul>
+      {tasks.map((task) => {
+        console.log("%cTodoList.jsx line:9 task", "color: #007acc;", task);
+        return <ListItem taskName={task.name} dueDate={task.deadline} />;
```

Comment:
You could just pass those props through `task={task}` and access them on the `ListItem.jsx` component.

Reply:
Thanks! Haven't thought about it! refactored the code accordingly!